primer/react
# feat(ToggleSwitch): add support for a ref on the inner <button> element

Closes #3806



### Changelog



#### New



#### Changed



- `ToggleSwitch` now accepts a `ref` on the inner `<button>` element
- Move ToggleSwitch test fixtures next to component

#### Removed



### Rollout strategy




### Testing & Reviewing



- Verify that `ToggleSwitch` supports `ref`
- Verify that tests capture semantics of `ref` (specifically as a HTMLButtonElement)
